src/AltinnCore/Common/Services/Implementation/ExecutionSIIntegrationTest.cs

```diff
@@ -244,5 +205,11 @@ public byte[] GetRuntimeResource(string resource)
 
             return fileContent;
         }
+
+        /// <inheritdoc/>
+        public void CheckAndUpdateWorkflowFile(string applicationOwnerId, string applicationId, string developer)
```

Comment:
When is this method called?

Reply:
Method is used to check if service is using the newest version of the workflow file, for now this is checked in runtime, but guessing it should be moved to the designer eventually since its only functionality that should be applied in altinn-studio not the app

Reply:
May be that can be a private method that can be called from another public method?

Reply:
Can't it be like this for now and then we move it when we know where the correct place to put it is?

Reply:
if it will take time we could do it later